NOTICE-binary

```diff
@@ -54,7 +54,6 @@ Copyright 2006-2019 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
-
```

Comment:
just pegging this in here, to make sure we're all on the same page: https://github.com/apache/flink/pull/9494#discussion_r316597961

Since the kinesis connector will be building and released with Flink once this PR is merged in, we should be adding the dependencies to the NOTICE-binary file, correct?

@zentol